pkg/apic/provisioning/requeststatus.go

```diff
@@ -0,0 +1,69 @@
+package provisioning
+
+import "fmt"
+
+const statusSetError = "can not set status as it's already been set"
+
+// RequestStatus - holds info about the status of the request
+type RequestStatus struct {
```

Comment:
Again this should be an interface with internal struct populated by the builder

Reply:
same